pkg/security/certManager/general/certManager_test.go

```diff
@@ -113,15 +113,15 @@ func createTmpCertFiles(t *testing.T, caFile, crtFile, keyFile string) general.C
 	require.NoError(t, err)
 
 	cfg := general.Config{
-		CAPool:   caFile,
+		CAPool:   []string{caFile},
 		KeyFile:  keyFile,
 		CertFile: crtFile,
 	}
 	return cfg
 }
 
 func deleteTmpCertFiles(t *testing.T, cfg general.Config) {
-	err := os.Remove(cfg.CAPool)
+	err := os.Remove(cfg.CAPool[0])
```

Comment:
I would range over the array here, that way the code will continue to work if we ever modify the constructor to create CAPool with multiple files.

Reply:
fixed